libs/ui/packages/react/src/components/layout/Carousel/Carousel.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Carousel from "./";
+
+const CarouselStory = () => {
```

Comment:
Can you add one story per variant, please? And connect the control to be able to switch from contentCard to default, please?